src/components/lib/address-link.tsx

```diff
@@ -0,0 +1,67 @@
+
+import Link from "next/link";
+import { SquareArrowOutUpRight } from "lucide-react";
+import { cn } from "@/lib/utils";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/components/ui/tooltip";
+import useGlobalContext from "@/components/lib/context/hook";
+
+interface AddressLinkProps {
```

Comment:
We lack a `className?: string` prop here to be able to configure the underlying `Link` style.